sample/simple-videoplayer-channel/components/MainScene.brs

```diff
@@ -347,13 +347,16 @@ sub onVideoPlayerStateChange()
   '--------------------
   ' Send Media events
   '--------------------
-  position = m.video_position
+
+  ' The playhead is the current position of the video in seconds.
+  ' It should be an integer value. Other values will be treated as an invalid playhead.
```

Comment:
It must be a non-negative integer value. Other values will be treated as invalid.